locales/en-US/messages.properties

```diff
@@ -73,6 +73,7 @@ mySixWordSummerTeachingKit=Six-Word Summer {{ teachingKit }}
 
 # Loading the editor
 loadingProject=Loading Project
+loadingProjectSupportSuggestion=If the project doesn't load, <a title="Thimble help page" href="https://support.mozilla.org/en-Us/products/webmaker/thimble"> get help here </a>.
```

Comment:
Please remove the spaces inside of the ``<a>`` tag. The trailing space is visible on the loading screen and looks a little off...

![image](https://cloud.githubusercontent.com/assets/25212/23238623/b7d05b32-f917-11e6-993a-6ce1e2090ab3.png)